.github/workflows/dashboard.yaml

```diff
@@ -13,4 +13,41 @@ jobs:
       with:
         java-version: 8
     - run: java -version
+    - run: .kokoro/dashboard.sh
+
+  converge:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'googleapis' && github.head_ref == 'release-please/branches/master'
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
+      script: |
+        // only approve PRs from release-please
+        if (context.payload.pull_request.user.login !== "release-please") {
+          return;
+        }
+
+        // only approve PRs like "chore: release <release version>"
+        if (!(/^chore: release.*$/.test(context.payload.pull_request.title))) {
+          return;
+        }
+
+        // only approve PRs with README.md, CHANGELOG.md, pom.xml and versions.txt changes
+        const files = new Set(
+          (
+            await github.paginate(
+              github.pulls.listFiles.endpoint({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                pull_number: context.payload.pull_request.number,
+              })
+            )
+          ).map(file => file.filename)
+        );
+        if (files.size != 4 || !files.has("README.md") || !files.has("CHANGELOG.md") || !files.has("pom.xml") || !files.has("versions.txt")) {
+          return;
+        }
+    - run: java -version
     - run: .kokoro/dashboard.sh
```

Comment:
if you are checking `JOB_TYPE` in `dashboard.sh`, you would need to add the env var here:
```
        env:
          JOB_TYPE: convergence
```